VRCFaceTracking.Core/AvatarConfigFileParser.cs

```diff
@@ -68,7 +68,7 @@ public AvatarConfigParser(ILogger<AvatarConfigParser> parserLogger)
         ParameterSenderService.Clear();
         var parameters = avatarConfig.parameters.Where(param => param.input != null).ToArray<IParameterDefinition>();
 
-        foreach (var parameter in UnifiedTracking.AllParameters_v2.Concat(UnifiedTracking.AllParameters_v1).ToArray())
+        foreach (var parameter in UnifiedTracking.AllParameters_v2.Concat(UnifiedTracking.AllParameters_v1).Concat(UnifiedTracking.HeadParameters).ToArray())
```

Comment:
Looks good, we probably should make this cleaner in the future  to support more addendums to the UE parameters